client/src/app/+about/about.component.html

```diff
@@ -10,4 +10,4 @@
     </ul>
     <p>הנתונים שמוצגים בפנסיה פתוחה לקוחים מהדו”חות המפורסמים ע"י הגופים המנהלים של קרנות הפנסיה. דו”חות אלו המציגים פירוט ברמת הנכס הבודד של הנכסים המוחזקים בכל קרן. הדו”חות מתפרסמים בסוף כל רבעון, ופנסיה פתוחה מעדכנת את מאגר הנתונים שלה לאחר מכן. לפנסיה פתוחה אין כל אחריות לנכונות ו/או לשלמות הנתונים המוצגים.&nbsp;</p>
     <p>בשלב זה, פנסיה פתוחה מציגה רק את הנתונים של חברת מגדל. בעתיד הקרוב, פנסיה פתוחה תקיף את כל שוק הפנסיה: קרנות פנסיה, קופות גמל וביטוחי מנהלים.</p>
-      
\ No newline at end of file
+  
```

Comment:
I think one more.
The problem is: You need to have a new line at the end of the file. But, GitHub says you don't have one (see the red arrow in the following image)
![add_text_to_about_page_by_eyalcohen4_ _pull_request__112_ _hasadna_open_pension](https://cloud.githubusercontent.com/assets/3472902/16268147/d7cfe0bc-3895-11e6-939c-3c5b45ef676f.png)

As you can see line 13 **is** empty, so I think GitHub say it because it's not empty and you have some whitespace there (spaces or tabs), it should be just a new line, with any spaces or any tabs.